designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+---
+parent: Adminstrator
+version: 4.6
+---
+
+# Cluster Settings 
+
+Users complete upgrades in cluster settings. The following design focuses on making improvements to the current upgrade experience in cluster settings by making upgrading as simple and transparent as possible. Specific improvements include adding more information around recommended upgrade paths and notifying users when new versions and channels become available. 
+
+## Adding Alerts to the Notification Drawer
+
+![Alerts - Available Updates](img/notifications-updates.png)
+- Today, available updates appear in the message section of the notification drawer. 
```

Comment:
My understanding is this will be "Recommendations" section in 4.6, I think this particular use is already on @cshinn radar.

Reply:
Yes, good catch. @megan-hall can you update to Recommendations, if @cshinn confirms that change for 4.6?

Reply:
very nitpicky but that icon is not aligned with the title :)

Reply:
Waiting on confirmation before I update :)

Reply:
Icon looks good now!